cmd/templates.go

```diff
@@ -0,0 +1,29 @@
+package dot
```

Comment:
Could you remove this file ? ( may be unnecessary )

Reply:
Oh,sorry.. I removed the file. Thanks!